swagger_templates/python/model.mustache

```diff
@@ -75,6 +75,35 @@ class {{classname}}(object):
         :type: {{datatype}}
         """
         {{#isEnum}}allowed_values = [{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]
+{{#isContainer}}
```

Comment:
All of these changes came as a result of merging the latest version of the upstream python/model.mustache with our version of it. Essentially, the latest version now tries to do validation of arrays and dictionaries, whereas in the past it just skipped it.